server/src/Domain/ValueObjects/FileMetadata.cs

```diff
@@ -0,0 +1 @@
+public record FileMetadata(string FileName, string FileType, long FileSize);
```

Comment:
Should be next to application Dtos, since we don't store file metadata in our database